Sharpmake/Project.Configuration.cs

```diff
@@ -1605,6 +1605,9 @@ public string FastBuildUnityPath
             /// </remarks>
             public bool DoNotGenerateFastBuild = false;
 
+            // Unity builds support
+            public int MaxFilesPerUnityFile = 0;
```

Comment:
You could maybe reuse the BlobCount 
```suggestion
            public int MaxFilesPerJumboFile = 0;
```